docs/OpenShift GitOps CLI User Guide.md

```diff
@@ -0,0 +1,373 @@
+# OpenShift GitOps CLI User Guide
+
+## Installing OpenShift GitOps CLI (argocd)
```

Comment:
@anandf Thanks for the doc inputs! Can we have a Configuring CLI section like this - https://docs.openshift.com/container-platform/4.10/cli_reference/tkn_cli/op-configuring-tkn.html and https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/configuring-cli.html?

Reply:
Added the configuration section.